### PR TITLE
Update user guide and GUI

### PR DESCRIPTION
**Pull request by jmleong666**
**Changes**
- Update archive commands in the user guide.
- Update icons on the Meetings and Sales list to change color according to the theme.

**Preview**
Check out the changes to UG here at https://jmleong666.github.io/tp/UserGuide.html. The updated light theme is shown below:
lightTheme

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -578,23 +586,30 @@ Shows the list of your archived contacts.
...
+**Example**:
+To see all your archived contacts, type `archive list` into the **Command Box** and press <kbd>Enter</kbd>. The **Result Box** will inform you that the command was successful, and the archived contacts list will appear.
 
-#### Removing contacts from the archive: `archive remove`
+#### Removing contacts from the archive: `archive remove` \[Leong Jin Ming\]
 
 Removes the specified contact from the archive. The specified contact will appear on your contact list again.
 
+<div markdown="block" class="alert alert-info">
+StonksBook will not remove a contact from the archive it is not in the archive!
```
Comment (sebastiantoh) [inline docs/UserGuide.md]:
```suggestion
StonksBook will not remove a contact from the archive if it is not in the archive!
```

File: src/main/resources/view/ReminderListCard.fxml [hunk 2]
```diff
@@ -27,11 +27,11 @@
         <Label fx:id="message" text="\$first" wrapText="true" styleClass="cell_big_label"/>
       </HBox>
       <HBox spacing="5" alignment="CENTER_LEFT">
-        <FontIcon fx:id="calendarIcon" iconLiteral="far-calendar-alt" iconColor="white" iconSize="16"/>
+        <FontIcon fx:id="calendarIcon" iconLiteral="far-calendar-alt" styleClass="big-icon"/>
```
Comment (sebastiantoh) [inline src/main/resources/view/ReminderListCard.fxml:30]: I think this sets the icon to be permanently white even in the dark theme. By right the icon should be green when the reminder is complete, and red when the reminder is overdue.

In the light theme, the text doesn't reflect the colours as well.

File: src/main/resources/view/MeetingListCard.fxml [hunk 3]
```diff
@@ -27,11 +27,11 @@
         <Label fx:id="message" text="\$first" wrapText="true" styleClass="cell_big_label"/>
       </HBox>
       <HBox spacing="5" alignment="CENTER_LEFT">
```
Comment (sebastiantoh) [inline src/main/resources/view/MeetingListCard.fxml:29]: Past meetings should be "greyed out" but this doesn't seem to be the case in the light theme

**Review by sebastiantoh — COMMENTED** [review]
LGTM! Think there's still some styling issue in the light theme (esp regarding reminders / meetings)

File: src/main/resources/view/ReminderListCard.fxml (hunk 2 above)
Comment (jmleong666) [reply src/main/resources/view/ReminderListCard.fxml:30]: I'll try to fix it asap!

File: src/main/resources/view/MeetingListCard.fxml (hunk 3 above)
Comment (jmleong666) [reply src/main/resources/view/MeetingListCard.fxml:29]: Styling issues should be fixed now. Let me know if there's still anything missing for the styling!

**Review by sebastiantoh — APPROVED** [review]
LGTM! Thanks for the fixes!

**Pull request merged**